turtlebot/a4f35ee/a4f35ee.bug

```diff
@@ -101,5 +101,5 @@ fix:
 fault-codes:
   - SYSTEMS:OS
 failure-codes:
-  - SOFTWARE:CRASHING
-  - SYSTEM:LOSS-OF-CONTROL
+  - SOFTWARE:BUILD
+  - SYSTEM:NONE
```

Comment:
After discussion at the meeting: Let's stick to "System: Loss of functionality."